quipucords/api/migrations/0032_add_ssh_keyvalue_to_credential.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.1 on 2023-06-07 22:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0031_alter_scantask_systems_count_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="credential",
+            name="ssh_keyvalue",
```

Comment:
given this migration does not exist on main branch, I'd change "ssh_keyvalue" right here - and purge migration 33 from git history 🔥 🔥 🔥

Reply:
done.